pkg/ingester/ingester_v2.go

```diff
@@ -420,13 +421,13 @@ func (i *Ingester) v2Query(ctx context.Context, req *client.QueryRequest) (*clie
 		return &client.QueryResponse{}, nil
 	}
 
-	q, err := db.Querier(int64(from), int64(through))
+	q, err := db.Querier(ctx, int64(from), int64(through))
 	if err != nil {
 		return nil, err
 	}
 	defer q.Close()
 
-	ss, err := q.Select(matchers...)
+	ss, _, err := q.Select(false, nil, matchers...)
```

Comment:
@pstibrany doesn't the querier expects sorted series?

Reply:
This is correct. Querier will need to merge different series from different ingesters anyway (based on fingerprint), and will then sort them.

Reply:
If it was coming sorted from ingester, querier would have easier time merging, but on the other hand, it would use more CPU time as well. We wan't to disturb ingesters as little as possible on read path.